tools/chocolateyInstall.ps1

```diff
@@ -1,8 +1,8 @@
 $packageName = 'vmwareworkstation'
 $installerType = 'exe'
-$url = 'https://download3.vmware.com/software/wkst/file/VMware-workstation-full-15.0.2-10952284.exe'
-$url64 = $url

```

Comment:
`$url64` is referenced [here](https://github.com/StefanScherer/choco-vmwareworkstation/blob/128b48aecd52b6dd981814929d4e1356ecca567f/tools/chocolateyInstall.ps1#L11) still.

Reply:
Oh, I accidentally removed it. The package still works, but I‘ll cleanup the script for next release. Thank you for finding this.